test/externalTests/gp2.sh

```diff
@@ -69,16 +69,6 @@ function gp2_test
     force_hardhat_unlimited_contract_size "$config_file" "$config_var"
     yarn
 
-    # New hardhat release breaks GP2 tests, and since GP2 repository has been archived, we are pinning hardhat
-    # to the previous stable version. See https://github.com/ethereum/solidity/pull/13485
-    yarn add hardhat@2.10.2
-    # hardhat-tenderly@1.2.0 and upwards break the build, hence we are pinning the version to the last stable one.
-    # See https://github.com/cowprotocol/contracts/issues/32
```

Comment:
Note that this issue was not fixed upstream, they only fixed the version to `1.1.6`. But I think we don't need to keep track of this here anymore.